internal/pkg/mapper/branchmetadata/local_load.go

```diff
@@ -17,5 +18,22 @@ func (m *branchMetadataMapper) MapAfterLocalLoad(recipe *model.LocalLoadRecipe)
 	}
 
 	branch.Metadata = manifest.MetadataMap()
+
+	if branch.IsDefault {
+		fileToLoad := recipe.Files.
+			Load(m.state.NamingGenerator().DescriptionFilePath(".")).
+			AddMetadata(filesystem.ObjectKeyMetadata, recipe.Key()).
+			SetDescription("project description").
+			AddTag(model.FileTypeMarkdown).
+			AddTag(model.FileKindProjectDescription)
+		file, err := fileToLoad.ReadFile()
+		if err != nil {
+			return err
```

Comment:
Tu by sa nemala kontrolovat chyba? Ze ak subor neexistuje, tak je to ok?

Reply:
^^^ @JakubMatejka to je to miesto, kde by sa mal ignorovat neexistujuci subor (IMHO)

Reply:
Jo, tady je chyba. Ale spíš teda to myslíš tak, že když v projektu není, tak nemám vytvářet prázdný?

Reply:
jj. 👍 já si zase popletl load a save 😅

Reply:
Viz. https://github.com/keboola/keboola-as-code/pull/698/commits/7dace5cd57a2558364a21ad5ae06ec9201405290 z validate testu jsem ten soubor odstranil aby bylo vidět že projde